<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirmwareUI</class>
 <widget class="QDialog" name="FirmwareUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设备名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>信息段起始地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>程序段起始地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设备地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>固件版本号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DeviceName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="DeviceAddr">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>31278</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="Version1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="InfoAddr">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>31278</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProAddr">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>31278</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="Version2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Version3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Version4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Version5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Version6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Version7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Version8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Sure">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>固件路径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TextFileAddr">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>FirmwareUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
